ai-assistants/chat/messages.ts

```diff
@@ -242,18 +302,54 @@ export const messageProcessorFor = async (
 
     if (!lastMsg) {
       // TODO(@mcandeia) in some cases the bot didn't respond anything.
+      const message: ReplyMessage = {
+        messageId: Date.now().toString(),
+        threadId: thread.id,
+        type: "error",
+        content: [],
+        role: "assistant",
+      };
+      reply(message);
       return;
     }
 
-    latestMsg = lastMsg.id;
-    reply(threadMessageToReply(lastMsg));
+    const replyMessage = threadMessageToReply(lastMsg);
+    console.log(
+      "functionCall length:",
+      functionCallReplies.length,
+    );
+
+    if (
+      functionCallReplies.length === 1 &&
+      functionCallReplies[0].name === "multi_tool_use.parallel"
+    ) {
+      const message: ReplyMessage = {
+        messageId: Date.now().toString(),
+        threadId: thread.id,
+        type: "error",
+        content: [],
+        role: "assistant",
+      };
+      reply(message);
+    } else {
```

Comment:
not sure why this is necessary. can you explain a bit?

Reply:
add a comment about it